extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -490,6 +490,10 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
      * @return {undefined}
      */
     filter(entry) {
+        if (!entry.value) {
```

Comment:
can't this happen with `delete` entries?

Reply:
We used to ignore these in the logReader, i moved that logic to the actual extensions as S3c uses true delete events for bucket notifications. In lifecycle we use the update we do before each delete as we need the metadata.